modules/stroeerCoreBidAdapter.js

```diff
@@ -220,13 +220,17 @@ export const spec = {
     }
 
     const gdprConsent = bidderRequest.gdprConsent;
-
     if (gdprConsent && gdprConsent.consentString != null && gdprConsent.gdprApplies != null) {
       commonPayload.gdpr = {
         consent: gdprConsent.consentString, applies: gdprConsent.gdprApplies
       };
     }
 
+    const globalKeyValues = getGlobalKeyValues();
```

Comment:
I think it's safe to set this directly on the `commonPayload` object like what you did with the local key values.
```
const commonPayload = {
   ...
   kvg: getGlobalKeyValues()
}
```

Reply:
ok. Done